src/libraries/Directory.Build.targets

```diff
@@ -226,9 +226,6 @@
   <!-- Adds Nullable annotation attributes to netstandard <= 2.0 builds -->
   <Choose>
     <When Condition="'$(Nullable)' != '' and ($(TargetFramework.StartsWith('netstandard1')) or '$(TargetFramework)' == 'netstandard2.0' or $(TargetFramework.StartsWith('netcoreapp2')) or '$(TargetFrameworkIdentifier)' == '.NETFramework')">
-      <PropertyGroup>
```

Comment:
Why are you removing this? This will break a lot of projects.

Reply:
This is fixing OOB Nullable attributed to be on the same plan as all other OOB attributes. https://github.com/dotnet/runtime/pull/39265#discussion_r454394027

Reply:
I see, so the plan is to use `SYSTEM_PRIVATE_CORELIB` instead?

Reply:
Right. It is what we are using in number of places already.